mergify/metadata.csv

```diff
@@ -1,2 +1,19 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 mergify.merge_queue_length,gauge,,,,Merge queue length,0,mergify,,
+mergify.time_to_merge.median,gauge,,second,,Median time to merge,0,mergify,,
+mergify.time_to_merge.mean,gauge,,second,,Mean time to merge,0,mergify,,
+mergify.queue_checks_outcome.PR_DEQUEUED,gauge,,,,The number of PR that have been manually removed from the queue,0,mergify,,
+mergify.queue_checks_outcome.PR_AHEAD_DEQUEUED,gauge,,,,The number of PR that have been removed from the queue because of a PR ahead of it was removed from the queue,0,mergify,,
+mergify.queue_checks_outcome.PR_AHEAD_FAILED_TO_MERGE,gauge,,,,The number of PR that have been removed from the queue because of a PR ahead of it failed to merge,0,mergify,,
```

Comment:
```suggestion
mergify.queue_checks_outcome.PR_AHEAD_FAILED_TO_MERGE,gauge,,,,The number of PRs that have been removed from the queue because a PR ahead of it failed to merge,0,mergify,,
```